tibiapy/enums.py

```diff
@@ -2,20 +2,36 @@
 
 
 class Sex(Enum):
-    FEMALE = "female"
     MALE = "male"
+    FEMALE = "female"
 
 
 class HouseStatus(Enum):
-    AUCTIONED = "auctioned"
     RENTED = "rented"
+    AUCTIONED = "auctioned"
 
 
 class HouseType(Enum):
     HOUSE = "house"
     GUILDHALL = "guildhall"
 
 
+class AccountStatus(Enum):
+    FREE_ACCOUNT = "Free Account"
+    PREMIUM_ACCOUNT = "Premium Account"
+
+
+class Vocation(Enum):
```

Comment:
You're missing one: https://secure.tibia.com/community/?subtopic=characters&name=sunrise